docker-entrypoint.sh

```diff
@@ -7,8 +7,8 @@ set_listen_addresses() {
 }
 
 if [ "$1" = 'postgres' ]; then
-	mkdir -p "$PGDATA"
-	chown -R postgres "$PGDATA"
+	mkdir -p "${PGDATA}"
+	chown -R postgres "${PGDATA}"
```

Comment:
yes, habit of mine and my editor highlights them the better, will take that back